Gems/Terrain/Code/Source/Components/TerrainWorldRendererComponent.cpp

```diff
@@ -42,6 +34,14 @@ namespace Terrain
                 ->Field("ClodDistance", &MeshConfiguration::m_clodDistance)
                 ;
 
+            serialize->Class<DetailMaterialConfiguration>()
+                ->Version(1)
+                ->Field("UseHeightBasedBlending", &DetailMaterialConfiguration::m_useHeightBasedBlending)
+                ->Field("RenderDistance", &DetailMaterialConfiguration::m_renderDistance)
+                ->Field("FadeDistance", &DetailMaterialConfiguration::m_fadeDistance)
+                ->Field("Scale", &DetailMaterialConfiguration::m_scale)
+                ;
+
```

Comment:
FYI, this didn't have to move to change the edit ordering, just the field in the TerrainWorldRendererConfig. No _harm_ in moving this though. :)

Reply:
yeah, did it on purpose for better consistency.